### PR TITLE
initial testall.py file with 3 basic test cases

### PR DESCRIPTION
**Pull request by ldwong20**

### CODE REVIEW

File: test/testall.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    # if it doens't match, throw an error describing what went wrong
+    filename = os.fsdecode(file)
+    if filename.endswith(".err"): 
+        file_ans_loc = os.path.join("./tests/", filename)
+        file_loc = os.path.join("./tests/", '.'.join(filename.split('.')[:-1] + ["bruh"]))
+  
+        result = subprocess.run(f"dune exec -- CoBruh -c {file_loc}".split(), stdout=subprocess.PIPE)
+        result = result.stdout.decode("utf-8") 
+
+        ans = ''
```
Comment (Akash-Nayar) [inline test/testall.py:20]: personal preference but i'd do `ans  = None` until we initialize it

File: test/testall.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        file_ans_loc = os.path.join("./tests/", filename)
+        file_loc = os.path.join("./tests/", '.'.join(filename.split('.')[:-1] + ["bruh"]))
+  
+        ir_result = subprocess.run(f"dune exec -- CoBruh -c {file_loc}".split(), stdout=subprocess.PIPE)
+        with open("main.ll", 'w') as writer:
+            writer.write(ir_result.stdout.decode("utf-8"))
+        result = subprocess.run(f"lli main.ll".split(), stdout=subprocess.PIPE)
+        result = result.stdout.decode("utf-8") 
+
+        ans = ''
```
Comment (Akash-Nayar) [inline test/testall.py:36]: ditto

**Review by 20joshuaz — APPROVED** [review]
you're very cute

**Review by Akash-Nayar — CHANGES_REQUESTED** [review]

**Comment by Akash-Nayar** [comment]
> you're very cute

@20joshuaz so true

**Review by Akash-Nayar — APPROVED** [review]

**Pull request merged**